desk/lib/vitals.hoon

```diff
@@ -11,6 +11,15 @@
 ++  scry-qos
   |=  [=ship =time peer=ship]
   ^-  qos:ames
+  ::  a ship is not guaranteed by %ames to know itself, so we fake it
+  ?:  =(ship peer)
+    [%live time]
+  ::  .^(* %ax /=//=/peers/[peer]) crashes if the peer is unknown, so we
+  ::  check the source map beforehand and fake an %unborn if we can see
+  ::  a crash coming
+  =/  peers  .^((map ^ship ?(%alien %known)) ~[%ax %$ %peers])
```

Comment:
The scry path here doesn't look right. `/ax//peers` isn't a valid scry path, you want to explicitly fill in the `=` parts from the comment. Scry helper functions might do that for you, but we're doing a raw `.^` here.

```hoon
.^((map ^ship ?(%alien %known)) /ax/(scot %p ship)//(scot %da time)/peers)
```

Reply:
Good catch... I was too eager in copy-pasting the code from the thread version. The `our` auto-filling is evidently something only done by `scry:strandio`, so I'll need to update this.